source/install/troubleshooting.rst

```diff
@@ -19,9 +19,41 @@ General Troubleshooting
 -----------------------
 Some of these suggestions can be done directly, and others may need consultation from your network administrator.
 
-- Take a look at the logs (``mattermost.log`` and NGINX logs) for errors.
-- You can also search the error messages online - existing solutions can often be applied.
-- Open **System Console > General > Logging** and set File Log Level to **DEBUG**. Make sure to revert to **INFO** after troubleshooting to save disk space.
+Review Mattermost Logs
+~~~~~~~~~~~~~~~~~~~~~~~
+
+You can access logs for Mattermost and use them for troubleshooting. These steps assume that you have `System Admin permissions <https://docs.mattermost.com/help/getting-started/managing-members.html#system-admin>`_. 
+
+**Mattermost Server**
+
+#. Verify that logging is enabled by navigating to **System Console > Environment > Logging** and confirming that **Output logs to file** is set to **true**.
+#. Navigate to **System Console > Environment > Logging** and note down the path to the **File Log Directory**.
```

Comment:
```suggestion
#. Navigate to **System Console > Environment > Logging** and note down the path to the **File Log Directory**.
```
The user should already be in this menu, as per the previous step, I wonder if we can restructure this as a single line and not a numbered list. Here's an example: 

"Navigate to **System Console > Environment > Logging**, confirm that **Output logs to file** is set to **true**, and note down the path to the **File Log Directory**." 

Then the next sentence will be "The server log file is called ``mattermost.log`` and can be opened with a standard text editor or shared directly."

Let me know what you think.

Reply:
I created it the redundant way so help people extract partial information from the doc. e.g. if I am solely looking for the log file path I'd have to read more. I'm 0/5 on this and I like your suggestion. WDYT @justinegeffen

Reply:
That makes sense. 

If it's two separate items then a bullet point for each would work better than steps as it's not a set process. Perhaps a preceding sentence for each might work as well. 

Like:
"- Ensure that log files are being created: Navigate to **System Console > Environment > Logging**, confirm that Output logs to file is set to **true**.

- You can obtain the path for the log files in **System Console > Environment > Logging > File Log Directory**. 

The resulting server log file is called ``mattermost.log`` and can be opened with a standard text editor or shared directly."

I know there's a section with the path to the log files. Perhaps add it here so it states:
"The resulting server log file is called ``mattermost.log``, located in <insert paths info> and can be opened with..."

I'm 3/5 as I don't have context around customer behavior in this instance. Totally up to you!

Reply:
sounds good, will do:

> - Ensure that log files are being created: Navigate to System Console > Environment > Logging, confirm that Output logs to file is set to true.
> - You can obtain the path for the log files in System Console > Environment > Logging > File Log Directory.
> 
> The resulting server log file is called mattermost.log and can be opened with a standard text editor or shared directly.